probe-rs/src/config/sequences/cc13xx_cc26xx.rs

```diff
@@ -306,6 +309,35 @@ impl CC13xxCC26xx {
     }
 }
 
+fn reset_chip(family: &str, probe: &mut dyn ArmProbe) -> Result<(), ArmError> {
+    // Do a full system reset (emulated PIN reset)
+    // CPU reset alone is not possible since AIRCR.SYSRESETREQ will be
+    // converted to system reset on these devices.
+    //
+    // The below code writes to the following bit
+    // AON_PMCTL.RESETCTL.SYSRESET=1
+
+    // The CC family of device have a pattern where the 6th character of the device name dictates the family
+    // Use this to determine the correct address to write to
+    match family.chars().nth(5).unwrap() {
+        '0' => {
+            probe.write_word_32(0x4009_0004, 0x8000_0000).ok();
+        }
+        '2' => {
+            probe.write_word_32(0x4009_0028, 0x8000_0000).ok();
+        }
+        '4' => {
+            probe.write_word_32(0x5809_0028, 0x8000_0000).ok();
+        }
+        _ => {
+            return Err(ArmError::Other(anyhow::anyhow!(
```

Comment:
We should probably try and return a more meaningful error type. Barring that, please include the name of the family that is unknown, in the error message. Or use `unreachable!()` with the same message with the family, as this is technically a programmer's error.

Reply:
You're right it wasn't really a good error, thing is that I was looking for `UnsupportedTarget` or something in `ArmError`, but I think it more sense to use unreachable and cleans things up a little.